manager/assets/modext/core/modx.layout.js

```diff
@@ -465,10 +475,13 @@ Ext.extend(MODx.Layout, Ext.Viewport, {
                     }
                 }
             });
-            buttons[i].onclick = function(e) {
+            var _this = this;
```

Comment:
`this` is already assigned to `el`, so making new variable `_this` is redundant, you can use `el`. 

Btw. I'm not a huge fan of passing scope this way, it can be completely avoided for example using arrow functions, but it's not the only case in this file where `this` is assigned, so I guess it doesn't matter that much right now.

Reply:
@theboxer thanks for the hint. I fix it 1c92247 ✅